pkg/auth/handler/webapp/authflowv2/login.go

```diff
@@ -184,6 +184,16 @@ func (h *AuthflowV2LoginHandler) ServeHTTP(w http.ResponseWriter, r *http.Reques
 		return nil
 	})
 
+	handlers.PostAction("ldap", func(s *webapp.Session, screen *webapp.AuthflowScreenWithFlowResponse) error {
+		serverName := r.Form.Get("x_servername")
+		serverName = "q_servername=" + serverName
```

Comment:
You never construct query like this. The query is NOT escaped properly. Use `url.Values` instead.